themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -31,7 +31,11 @@
 <?php if ($hasSubLinks): ?>
   <div class="<?=implode(' ', $classList) ?>">
 <?php else: ?>
-  <a href="<?=$toggleUrl ?>" class="<?=implode(' ', $classList) ?>" data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?> data-lightbox-ignore>
+  <a href="<?=$toggleUrl ?>"
+     class="<?=implode(' ', $classList) ?>"
+     data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>"
+     data-count="<?=$this->facet['count'] ?>"
+     <?php if($this->facet['isApplied']): ?>title="<?=$this->transEscAttr('applied_filter') ?>:"<?php endif;?> data-lightbox-ignore>
```

Comment:
I definitely appreciate the improved readability here, but I wonder if the style might need a little adjusting for consistency. This is using a four-space indent instead of a two-space indent (which might be justified, since it makes the attribute names align... but it's a little different from everything else). Maybe it would help to put the closing `>` character de-indented on a line by its own to make it more clear at a glance that this is all one big tag. @crhallberg might have thoughts on this too.

Another alternative might be to refactor the code to use the `htmlAttributes` view helper.

Reply:
Oh, I just did it for readability and forgot to mush it all back together.

Reply:
Thanks. I think it would be a good idea for us to rewrite a number of templates using `htmlAttributes` and other helpers for increased long-term readability, but I guess that's better handled as a separate PR so it doesn't mix multiple unrelated tasks into a single commit.

Now that this is a single line it's easier to compare what you changed, which leads me to two comments:

1.) You've moved the space from in front of the "title" attribute to be in front of the "<?php if" statement associated with inserting a title. I think it was better where it was; otherwise, there will be an extra unnecessary space in the tag when title is not displayed.

2.) You've added a colon onto the end of the title. I think it would be better to include the punctuation in the language files rather than hard-coding it into the template, so it's easier to apply i18n. Maybe it's worth creating a new applied_filter_label or applied_filter_title translation string that just copies the existing string but adds the colon. It should be possible to automate this with the command line language tools. If you agree with the approach but don't know how to use the tools, I'm happy to help with this step.

Reply:
It should be resolved to be back the way it was + the colon in the language files. I'd rather not mess with the command line as I'm too inexperienced with this.

Reply:
The reason I had suggested creating a new string for the version with the colon is that it's easier to remember to get translation strings for all the languages when we introduce new strings than it is to keep track of changes when we edit existing strings. Of course, if `applied_filter` is only used in one place in all the code, then we might as well just edit it in place instead of introducing something new. Have you done that analysis yet, or would you like me to dig into it when time permits?

Reply:
It is only used in single-facets.phtml. But of course people might have used it elsewhere in their installations ...

Reply:
I've added the colon to all the language files. We can easily undo this if we change our minds -- is it appropriate for a title attribute to end in a colon?

Reply:
Yes, as far as I know, it is perfectly fine.